File: src/main/java/com/googongill/aditory/controller/dto/link/request/UpdateLinkRequest.java
package com.googongill.aditory.controller.dto.link.request;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class UpdateLinkRequest {
    @NotBlank
    private String title;
    @NotBlank
    private String summary;
    @NotBlank
    private String url;
    @NotNull
    private Long categoryId;
}

File: src/main/java/com/googongill/aditory/controller/dto/link/request/UpdateLinkRequest.kt
package com.googongill.aditory.controller.dto.link.request

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull

data class UpdateLinkRequest (
    @NotBlank
    val title: String,
    @NotBlank
    val summary: String,
    @NotBlank
    val url: String,
    @NotNull
    val categoryId: Long
)
